scripts/databricks_install.sh

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# ---------------------------------------------------------
+# This script installs Recommenders into Azure Databricks
+
+echo "Preparing Recommenders library file (egg)..."
+zip -r -q Recommenders.egg . -i *.py -x tests/\* scripts/\*
+
+echo "Uploading to databricks..."
+dbfs cp --overwrite Recommenders.egg dbfs:/FileStore/Recommenders.egg
```

Comment:
typically when you use the UI, this is placed under dbfs:/FileStore/jars/<hash>-Recommenders.egg

maybe it's ok to exclude the hash info if the goal is to overwrite it with the current version, but maybe it's a good idea to put it in the jars folder too?

Reply:
Yeah, I intentionally removed 'jars' path to distinguish between the libs uploaded via UIs and this, but make sense to put it back since we don't need to distinguish them. I will put  jars path back.

For hash, yes, the goal is to overwrite so let's not use that.

Reply:
Changed